### DISCUSSION TITLE
提供一个好用的版本 需要的拿走不谢

### DISCUSSION THREAD

**Discussion by jean-reno** [getlantern/lantern-forum-cn#328] [General]
不用卸载 直接安装202-01-18日的版本 就能正常使用 退出后就不行了 要是再想用只能再安装一次 静等官方修复

**Comment by channsguo** [comment]
帮忙补充，这里面的BR2下载下来然后解压，放到lantern文件夹内，打开lantern可用
https://github.com/getlantern/lantern/releases/tag/7.7.2

**Reply by jean-reno** [reply]
下载哪个呢？

**Reply by channsguo** [reply]
@jean-reno 如果你是WIN，就win_x64.BR2

**Reply by channsguo** [reply]
解压出来放进去LANTERN的安装文件夹

**Comment by eryijiu025** [comment]
以测可行感谢

**Comment by Ph750218** [comment]
然后用手机专业版（还可以正常使用）激活PC的专业版，PC版会自动升级到7.8.1版，现在大概恢复了，使用体验中...

**Comment by Ph750218** [comment]
猜测大概率是PC版的7.8.3版没有充分测试就放出来了。
